docs/tasksDir/psp-migration.md

```diff
@@ -392,3 +392,296 @@ The following table show which Kubewarden policy can be used to replace each PSP
 | [forbiddenSysctls](https://kubernetes.io/docs/concepts/security/pod-security-policy/#apparmor)                | [sysctl-psp-policy ](https://github.com/kubewarden/sysctl-psp-policy ) |
 | [allowedUnsafeSysctls](https://kubernetes.io/docs/concepts/security/pod-security-policy/#apparmor)            | [sysctl-psp-policy ](https://github.com/kubewarden/sysctl-psp-policy ) |
 
+## PSP migration script
+
+The Kubewarden team has written a script that leverages the migration tool written
+by people from [AppVia](https://github.com/appvia/psp-migration), to migrate PSP
+automatically. With this script the PSP will be migrated for the equivalent
```

Comment:
So the AppVia people have written a script that we have modified to suit our use case? In which case we need to state what their script does.
```suggestion
automatically. With this script, the PSP will be migrated for the equivalent
```

Reply:
Done, please review it again.